### PR TITLE
x-ghaf-hw-test changes

### PR DESCRIPTION
**Pull request by vjuntunen**
Flashing and boot test can be skipped
if alreeady flashed and boot tested
with that same image

Signature verification is commented out for now.

Add branch parameter for ci-test-automation
repo

Remove commented line from
ghaf-parallel-hw-test.groovy

### CODE REVIEW

File: tests/x-ghaf-hw-test.groovy [hunk 1]
```diff
@@ -155,21 +190,22 @@ pipeline {
...
           // env.IMG_WGET stores the path to image as downloaded from the remote
           env.IMG_WGET = run_wget(params.IMG_URL, TMP_IMG_DIR)
           println "Downloaded image to workspace: ${env.IMG_WGET}"
           // Verify signature using the tooling from: https://github.com/tiiuae/ci-yubi
-          sig_path = run_wget("${params.IMG_URL}.sig", TMP_IMG_DIR)
-          println "Downloaded signature to workspace: ${sig_path}"
-          sh "nix run github:tiiuae/ci-yubi/bdb2dbf#verify -- --path ${env.IMG_WGET} --sigfile ${sig_path}"
+          // sig_path = run_wget("${params.IMG_URL}.sig", TMP_IMG_DIR)
+          // println "Downloaded signature to workspace: ${sig_path}"
+          // sh "nix run github:tiiuae/ci-yubi/bdb2dbf#verify -- --path ${env.IMG_WGET} --sigfile ${sig_path}"
```
Comment (azbeleva) [inline tests/x-ghaf-hw-test.groovy:208]: commented code

File: tests/x-ghaf-hw-test.groovy [hunk 2]
```diff
@@ -45,17 +83,23 @@ def get_test_conf_property(String file_path, String device, String property) {
...
+    env.INCLUDE_TEST_TAGS = "${test_tags}AND${env.DEVICE_TAG}"
+    println "Run BOOT test with these tags: -i ${env.INCLUDE_TEST_TAGS}"
   } else {
-    env.INCLUDE_TEST_TAGS = "${testname}AND${env.DEVICE_TAG}"
+    if (test_tags) {
+      env.INCLUDE_TEST_TAGS = "${test_tags}"
+      println "Run test with these tags: -i ${test_tags}"
+    } else {
+      println "Test tags is empty, give test tags as parameter when build this job!"
+    }
```
Comment (azbeleva) [inline tests/x-ghaf-hw-test.groovy:102]: not clear if it is possible to run boot test AND in case if it passes run bat tests?

Comment (vjuntunen) [reply tests/x-ghaf-hw-test.groovy:102]: Boot test is default if not skipped with 'FLASH' parameter 'false'. User input tags only affects after boot test passed OR if boot test is skipped.

File: tests/x-ghaf-hw-test.groovy [hunk 3]
```diff
@@ -155,21 +190,22 @@ pipeline {
             echo { \\\"Job\\\": \\\"${env.TARGET}\\\" } > ${TEST_CONFIG_DIR}/${BUILD_NUMBER}.json
             ls -la ${TEST_CONFIG_DIR}
           """
-          env.TESTSET = params.getOrDefault('TESTSET', '_boot_')
-          println "Using TESTSET: ${env.TESTSET}"
+          // env.TESTSET = params.getOrDefault('TESTSET', '_boot_')
+          // println "Using TESTSET: ${env.TESTSET}"
```
[the remark this answers is not in the document]
Comment (vjuntunen) [reply tests/x-ghaf-hw-test.groovy:194]: Yes I'll remove those.

**Review by azbeleva — APPROVED** [review]

**Pull request merged**